analysis/warriorfury/src/modules/shadowlands/legendaries/NaturesFury.tsx

```diff
@@ -0,0 +1,57 @@
+import SPELLS from 'common/SPELLS';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+class NaturesFury extends Analyzer {
+  _ancientAftershockDamage: number = 0;
+  _naturesFuryDamage: number = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active =
+      this.selectedCombatant.hasCovenant(COVENANTS.NIGHT_FAE.id) &&
+      (this.selectedCombatant.hasLegendary(SPELLS.NATURES_FURY) ||
+        this.selectedCombatant.hasLegendary(SPELLS.UNITY));
+
+    this.addEventListener(
+      Events.damage.by(SELECTED_PLAYER).spell(SPELLS.ANCIENT_AFTERSHOCK),
```

Comment:
you can do [SPELLS.ANCIENT_AFTERSHOCK, SPELLS.ANCIENT_AFTERSHOCK_DOT] to save some space 
iirc